pkg/proto/datadog/trace/idx/tracer_payload.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package datadog.trace.idx;
+
+option go_package="github.com/DataDog/datadog-agent/pkg/proto/pbgo/trace/idx";
+
+import "datadog/trace/idx/span.proto";
+
+// TraceChunk represents a list of spans with the same trace ID. In other words, a chunk of a trace.
+message TraceChunk {
+	// priority specifies the string table ref of the sampling priority of the trace.
+	int32 priority = 1;
+	// originRef specifies the string table ref of the origin product ("lambda", "rum", etc.) of the trace.
+	uint32 originRef = 2;
+	// attributes specifies the map of string table ref to AnyValue of the tags common in all `spans`.
+	map<uint32, AnyValue> attributes = 3;
+	// spans specifies list of containing spans.
+	repeated Span spans = 4;
+	// droppedTrace specifies whether the trace was dropped by samplers or not.
+	bool droppedTrace = 5;
+	// traceID specifies the ID of the trace to which all spans in this chunk belong.
+	bytes traceID = 6;
+	// samplingMechanism specifies the optional sampling mechanism (previously part of span tag _dd.p.dm)
+	uint32 samplingMechanism = 7;
+}
+
+// TracerPayload represents a payload the trace agent receives from tracers.
+message TracerPayload {
+	// strings specifies the array of strings referenced in this tracer payload, its chunks and spans.
+	repeated string strings = 1;
+	// containerIDRef specifies the string table ref of the ID of the container where the tracer is running on.
+	uint32 containerIDRef = 2;
```

Comment:
Would it be worth adding the `containerTags` here as well, or do you intend to place them somewhere else? I like the idea of having a dedicated field for them.